docs/sources/reference/components/discovery/discovery.process.md

```diff
@@ -187,6 +190,34 @@ discovery.process "all" {
   }
 }
 
+### Example discovering processes on the local host based on `cgroups` path
+
+To discover processes running under systemd services on the local host.
```

Comment:
```suggestion
The following example configuration shows you how to discover processes running under systemd services on the local host.
```

We could expand this intro sentence a little (I know the previous example doesn't have this intro sentence... fixing things like this are on the list of things to improve in the docs).